### ISSUE TITLE
Strange behavior in settings and selection of subtitles Android TV - remote control

### ISSUE THREAD

**Issue by walterdure** [rereleased/release#9]
In the subtitle color selection options, there is no indicator of which position is selected. You have to try randomly to choose the right color.
And playing any video, when you want to select subtitles, it doesn't normally scroll up and down, you have to go from left to right, return to the column and just scroll, although sometimes it doesn't work either.
Selecting subtitles from the internet most of the time does not respond, you have to cancel, re-enter the selection menu, scroll to the right to the bottom and then go up so that it takes the selection and starts searching.

**Comment by Morgan-le-Fay** [comment]
Yes the whole settings menu, logging into open subtitles, sync subtitle - all these UI are broken on AndroidTV/FireTV. I made an issue in the old repo but there was no engagement.
